src/nashpy/algorithms/regret_minimization.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+from typing import Generator, Tuple, Any
+import numpy.typing as npt
+
+
+def compute_regrets(strategy_utilities, current_strategy):
+    """
+    This function calculates the regrets for a player based on the strategy utilities and the current strategy.
+    Regrets represent the difference between the utility achieved by playing a strategy and the maximum utility
+    that could have been achieved by playing any strategy.
+    In this implementation, only positive regrets are considered
+    """
+    regrets = np.maximum(0, strategy_utilities - current_strategy)
+    return regrets
+
+
+def update_strategy(current_strategy, regrets, learning_rate):
+    """
+    This function updates the player's strategy based on the regrets, the current strategy, and a fixed learning rate.
+    It scales the regrets by the learning rate and adds them to the current strategy.
+    Finally, it normalizes the updated strategy to ensure that the probabilities sum up to 1.
+    """
+    updated_strategy = current_strategy + learning_rate * regrets
+    return updated_strategy / np.sum(updated_strategy)
+
+
+def generate_abs_strategy(strategy_list):
+    """
+    This function will return most favorable utility by a player based on the max probability value
+    """
```

Comment:
All the docstrings will need to match the numpy docstring style, which is checked in the CI using darglint: https://nashpy.readthedocs.io/en/stable/contributing/how-to/how-to-write-a-docstring/index.html. (More information here: https://nashpy.readthedocs.io/en/stable/contributing/discussion/darglint/index.html)

Reply:
The docs are matched and the tox ran successfully within my machine